src/managers/GuildChannelManager.js

```diff
@@ -60,21 +60,26 @@ class GuildChannelManager extends BaseManager {
    */
 
   /**
-   * Creates a new channel in the guild.
-   * @param {string} name The name of the new channel
-   * @param {Object} [options] Options
-   * @param {string} [options.type='text'] The type of the new channel, either `text`, `voice`, `category`, `news`,
+   * Options used to create a new channel in a guild.
+   * @typedef {Object} GuildCreateChannelOptions
```

Comment:
Maybe name this `GuildChannelCreateOptions` similar to the emoji one below?

Reply:
Done